charts/logging-apps/values.yaml

```diff
@@ -18,7 +18,7 @@ loki:
   # -- Chart
   chart: "loki"
   # -- [loki Helm chart](https://github.com/grafana/loki/tree/main/production/helm/loki)
-  targetRevision: "6.32.0"
+  targetRevision: "6.38.0"
```

Comment:
maybe we could bump to latest as of today?

Reply:
DOne!